src/utils/context.rs

```diff
@@ -80,3 +82,13 @@ impl<'r> FromRequest<'r> for RequestContext {
         });
     }
 }
+
+impl<'a> OpenApiFromRequest<'a> for RequestContext {
```

Comment:
Could you add a comment what this is doing?

Reply:
I agree with RIchard here. The comment is a little bit confusing.

From my understanding on how this framework works:
- For the routes that you want to have swagger enabled you need to have `#[openapi]` macro
- For those same routes, the framework needs to know how to represent custom types that implement `FromRequest` (in this case,` RequestContext`)
- By setting mapping  `RequestContext` to `RequestHeaderInput::None` we tell the framework that no user input is required for the header generation (thus it won't show in the swagger API)

Reply:
@moisses89 I think it's better to improve the documentation for this implementation for future reference. What do you think?